src/components/authorization/policies/by-role/consultant-manager.policy.ts

```diff
@@ -26,6 +26,14 @@ import { member, Policy, Role, sensMediumOrLower, sensOnlyLow } from '../util';
   r.Partnership.read.specifically((p) => [
     p.many('partner', 'organization').whenAny(member, sensOnlyLow).read,
   ]),
+  r.ProgressReport.whenAny(member, sensMediumOrLower).specifically(
+    (p) => p.status.read,
+  ),
```

Comment:
I think we want to be able to read the object/any property, right? In that cast it would be this:
```suggestion
  r.ProgressReport.whenAny(member, sensMediumOrLower).read,
```